src/Input/Textarea/Textarea.test.tsx

```diff
@@ -121,14 +123,19 @@ describe('<Textarea /> forwards ref to underlying textarea element', () => {
 
   test('ref is being forwarded correctly', () => {
     const ref = React.createRef<HTMLTextAreaElement>();
-    const { queryByLabelText } = render(<Textarea ref={ref} label={label} />);
+    const { queryByLabelText } = render(
+      <Textarea ref={ref} label={label} value={props.value} />
+    );
     const textarea = queryByLabelText(label);
     expect(textarea).toEqual(ref.current);
   });
 
   test('underlying textarea can be focused/blurred through ref', () => {
     const ref = React.createRef<HTMLTextAreaElement>();
-    const { queryByLabelText } = render(<Textarea ref={ref} label={label} />);
+    const fn = jest.fn();
+    const { queryByLabelText } = render(
+      <Textarea ref={ref} label={label} onBlur={fn} />
```

Comment:
For `onBlur` we should do something like `expect(onBlur).toHaveBeenCalledTimes(1);`